src/locales/pl.json

```diff
@@ -205,6 +232,7 @@
       },
       "reset": "Resetuj",
       "seal": "Pieczęć",
+      "set_as_default": "Set as default",
```

Comment:
👀  @javnik36 while we are at it?

Reply:
<img width="313" alt="Screenshot 2025-06-06 at 08 59 59" src="https://github.com/user-attachments/assets/a1a0d214-eecb-422b-938a-24770f8fc173" />